src/herder/HerderImpl.cpp

```diff
@@ -1187,9 +1187,14 @@ HerderImpl::getJsonQuorumInfo(NodeID const& id, bool summary, bool fullKeys,
     ret["node"] = mApp.getConfig().toStrKey(id, fullKeys);
     ret["qset"] = getSCP().getJsonQuorumInfo(id, summary, fullKeys, index);
     bool isSelf = id == mApp.getConfig().NODE_SEED.getPublicKey();
-    if (isSelf && mLastQuorumMapIntersectionState.hasAnyResults())
+    if (isSelf)
     {
-        ret["transitive"] = getJsonTransitiveQuorumIntersectionInfo(fullKeys);
+        ret["lag_ms"] = getHerderSCPDriver().getQsetLagInfo(summary, fullKeys);
```

Comment:
this should be under the `qset` node.

Also, you need to update the `admin.md` file that contains examples (both compact via `info` and normal)